src/main/java/seedu/address/logic/commands/AddCommand.java

```
@@ -62,6 +63,10 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_DUPLICATE_PERSON);
         }
 
+        if (model.hasPolicyNumber(toAdd)) {
+            throw new CommandException(MESSAGE_USED_POLICY_NUMBER);
```

Nice that you used the `throw` keyword rather than the usual `return` when handling exceptions that doesn't use `try`